src/app/features/bookshelf/bookcard.component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container>
```

Comment:
Why do we need this ng-container element? What happened if you remove it?

Reply:
Nothing happens, also I delete the one  on the bookcard